org.standardnotes.standardnotes.yml

```diff
@@ -0,0 +1,149 @@
+  - '--talk-name=org.freedesktop.secrets'
+modules:
+  - shared-modules/python2.7/python-2.7.json
+  - shared-modules/libsecret/libsecret.json
+  - name: yarn
+    buildsystem: simple
+    build-commands:
+      - 'cp -a * /app'
+    # Only used for building, so clean it up afterwards.
+    cleanup:
+      - '*'
+    sources:
+      - type: archive
+        url: https://github.com/yarnpkg/yarn/releases/download/v1.22.10/yarn-v1.22.10.tar.gz
+        sha256: 7e433d4a77e2c79e6a7ae4866782608a8e8bcad3ec6783580577c59538381a6e
+        x-checker-data:
+          type: anitya
+          'project-id': 13363
+          tag-template: https://github.com/yarnpkg/yarn/releases/download/$version/yarn-$version.tar.gz
+  - name: standardnotes
+    buildsystem: simple
+    build-options:
+      # Add the node bin directory.
+      append-path: '/usr/lib/sdk/node14/bin:/app/yarn/bin:/run/build/standardnotes/flatpak-node/chromedrive'
+      env:
+        # Don't add ELECTRON_CACHE
+        XDG_CACHE_HOME: /run/build/standardnotes/flatpak-node/cache
+        npm_config_nodedir: /usr/lib/sdk/node14
+        npm_config_offline: 'true'
+        npm_config_no_save: 'true'
+        npm_config_cache: /run/build/standardnotes/flatpak-node/npm-cache
+        npm_config_loglevel: 'verbose'
+    build-commands:
+      # No need to prettify the code
+      - sed -i -e "s/npm run lint:formatting && npm run lint:eslint && //"
+        package.json
+      # put --offline option to yarn
+      - sed -i 's/yarn/yarn --offline/g' package.json
+      - sed -i 's/yarn/yarn --offline/g' web/package.json
+
+      - cp .env.js.sample .env.js
+
+      # Use the offline mirror.
+      - 'HOME=$PWD yarn config --offline set yarn-offline-mirror
+        $FLATPAK_BUILDER_BUILDDIR/flatpak-node/yarn-mirror'
+
+      # Translated from npm run setup
+      - npm install
+      - npm --prefix ./app install
+      - yarn --offline --pure-lockfile --cwd ./web
+
+      - npm --offline run bundle
+      - . flatpak-node/electron-builder-arch-args.sh &&
+        ./node_modules/.bin/electron-builder --linux $ELECTRON_BUILDER_ARCH_ARGS
+        --dir
+      # Copy the resulting, unpacked directory to /app.
+      - 'cp -r dist/linux-unpacked /app/standardnotes'
+
+      # Install the wrapper script to start it.
+      - 'install -Dm 755 start-standardnotes.sh /app/bin/start-standardnotes'
+
+      # Retire this at some point when an SVG is available
+      - install -Dm644 app/icon/Icon-256x256.png
+        /app/share/icons/hicolor/256x256/apps/org.standardnotes.standardnotes.png
+      - install -Dm644 app/icon/Icon-512x512.png
+        /app/share/icons/hicolor/512x512/apps/org.standardnotes.standardnotes.png
+    sources:
+      # TODO: Replace with the git source
+      # https://github.com/standardnotes/web/pull/516
+      - type: archive
+        sha256: baf51d6cfac760d8974faeff3256ddb9478ba363059d99ae3b61cd8b6c569825
+        url: https://github.com/standardnotes/desktop/archive/v3.5.18.tar.gz
```

Comment:
Why not use git source here? I guess you won't need to add standardnotes/web tarball separately, then?